docs/ldml/tr35-keyboards.md

```diff
@@ -2229,7 +2229,7 @@ _Attribute:_ `from` (required)
 
 - **Nested capturing groups**
 
-    `(?:[abc]([def]))|(?:[ghi])`
+    `(?:[abc] ([def]))|(?:[ghi])`
```

Comment:
Is the added space intentional?

Reply:
it looks like it was interpreted as a markdown link!